core/src/util.jl

```diff
@@ -709,3 +709,19 @@ function get_influx(basin::Basin, basin_idx::Int)::Float64
     return precipitation[basin_idx] - evaporation[basin_idx] + drainage[basin_idx] -
            infiltration[basin_idx]
 end
+
+function get_discrete_control_indices(discrete_control::DiscreteControl, condition_idx::Int)
+    (; greater_than) = discrete_control
+    condition_idx_now = 1
+
+    for (compound_variable_idx, vec) in enumerate(greater_than)
```

Comment:
This method can use a docstring and/or some inline documentation on why/what is happening.

Reply:
This function is already removed in the follow up PR https://github.com/Deltares/Ribasim/pull/1393.